selectors.css

```diff
@@ -0,0 +1,78 @@
+/*The header element should have a 1px border. Color, you pick.*/
+
+header {
+	border: 1px solid #51297b;
+}
+
+/*Convert the ul in the navigation element into a series of horizontal links with # as the href value, without bullets, and have some space between them horizontally.*/
+/*Ensure that the navigation is semantically marked as such (i.e. wrap it in the correct HTML tag).*/
+
+nav ul * {
+	display: inline-block;
+	margin-right: 30px;
+}
+
+/*Any text in an element with class "disabled" should be colored grey, unless it is inside an anchor tag. If inside an anchor, it should be colored purple.*/
+
+.disabled {
+	color: grey;
+}
+
+a.disabled {
+	color: purple;
+}
+
+/*Any text inside an element with a class of "active" should be colored yellow.*/
+
+.active {
+	color: yellow;
+}
+
+/*Section elements should be contained within an article element.*/
+/*There are two missing closing tags in this document. Make sure you add them back in.*/
+
+/*Make the "I'm red" text colored red*/
+
+.parent div:first-child {
```

Comment:
Since you're using `.parent` in this and subsequent lines, you can wrap everything under `.parent` to improve readability